### PR TITLE
adding a ZD ticket comment to the github issue template

### PR DESCRIPTION
**Pull request by dani-tweig**
Updating the github issue template.

Fixes https://github.com/scylladb/scylla-enterprise/issues/3950

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.
 Scylla version (or git commit hash):
 Cluster size:
 OS (RHEL/CentOS/Ubuntu/AWS AMI):
+ZD ticket:
```
Comment (mykaul) [inline .github/ISSUE_TEMPLATE.md:11]:
```suggestion
ZenDesk ticket:
```

**Comment by mykaul** [comment]
If we are already touching the template, a link to the backoffice (in case of a Scylla cloud cluster) would be greatly appreciated as well.

**Comment by avikivity** [comment]
This open-source project is not managed with zendesk.

**Comment by nyh** [comment]
> This open-source project is not managed with zendesk.

Not only that, most of us don't even have read permissions to those links. So I think it's simply not relevant.

**Comment by denesb** [comment]
@dani-tweig please open this PR against the scylla-enterprise repository.

**Pull request closed without merging**